From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 29 Aug 2022 15:31:52 -0400
Subject: [PATCH] Work around GCC being obnoxiously incompatible with GCC

GCC added and then later removed the diagnostic flag
"-Wanalyzer-use-of-uninitialized-value", and so this doesn't work with
newer versions of GCC.

This patch removes the previous workaround for when it didn't work well.
I really wish any of our compilers had any sense of rigor with this
stuff at all.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/daemon.c | 5 -----
 1 file changed, 5 deletions(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
